pipeline_dp/mean_var_params.py

```diff
@@ -0,0 +1,132 @@
+"""DP computing of count, sum, mean, variance."""
+
+import numpy as np
+import pipeline_dp
+
+from dataclasses import dataclass
+
+
+@dataclass
+class MeanVarParams:
+    """Manages the parameters used for computing the mean/variance."""
+    eps: float
+    delta: float
+    low: float
+    high: float
+    max_partitions_contributed: int
+    max_contributions_per_partition: int
+    noise_kind: pipeline_dp.NoiseKind  # Laplace or Gaussian
+
+    def l0_sensitivity(self):
```

Comment:
I like this approach with having l0 sensitivty function (to emphasize how l0 sensitivity is computed from the parameters that an API user specified